memorix-recordtypes/Bestanddeel.ttl

```diff
@@ -118,6 +118,15 @@ rt:Bestanddeel
                                       sh:or        ( [ sh:class rt:SaaArchiefblok ] [ sh:class rt:SaaGroep ] ) ;
                                       sh:order     1.0 ;
                                       sh:path      rico:isOrWasIncludedIn ] ;
+    sh:property                     [ rdfs:label   "Volgende archiefbeschrijving in de volgorde van de archiefbeschrijvingen"@nl ;
+                                      rdfs:comment "Groep of Bestanddeel die op dit Bestanddeel volgt."@nl ;
+                                      sh:message   "Kies een groep of bestanddeel om aan te koppelen. Dit is verplicht."@nl ;
+                                      dash:editor  memorix:LinkedRecordEditor ;
+                                      sh:group     bestanddeel:partitionGroup ;
+                                      sh:path      rico:precedesOrPreceded ;
```

Comment:
Ivo, zoals al eerder gezegd weet ik nog niet OF we dit nodig hebben (ik wacht even op de uitwerking ervan in Memorix) en ik denk dat als we het nodig hebben, het logischer zou zijn dat we hier de inverse property zouden gebruiken: rico:followsOrFollowed.
Er even vanuitgaand dat de meeste beschrijvers nieuwe stukken telkens zullen toevoegen **na** eerdere stukken die ze al beschreven hebben en veel minder vaak andersom? Dan is het hadniger dat ze bij een stuk kunnen aangeven na welk ander stuk het komt, denk ik.

Reply:
Goede punten. Laten we afwachten en deze pull request voor nu afwijzen.